tests/test_s3.py

```diff
@@ -619,6 +620,24 @@ def test_attach_encrypt_requires_role(self):
              'resource': 's3',
              'actions': [{'type': 'attach-encrypt'}]})
 
+    @skip_if_not_validating
```

Comment:
The reason to skip is asymmetry between Travis and Drone. We [set](https://github.com/capitalone/cloud-custodian/blob/afe83481a95422df306fc7ef71a355b2069029fd/.travis.yml#L6) `C7N_VALIDATE=yes` at the former but [not](https://github.com/capitalone/cloud-custodian/blob/afe83481a95422df306fc7ef71a355b2069029fd/.drone.yml#L5-L8) at the latter. See https://github.com/whit537/cloud-custodian/issues/1#issuecomment-279134438 ff. Alternative would be to standardize test environments one way or another.